UPGRADE-2.13.md

```diff
@@ -0,0 +1,13 @@
+UPGRADE FROM 2.12 to 2.13
+========================
+
+Configuration
+-------------
+
+### Controller resolver auto mapping deprecated
+
+The controller resolver auto mapping functionality has been deprecated with Symfony 7.1, and is replaced with explicit mapped route parameters. Enabling the auto mapper using this bundle is now deprecated as well. The configuration option will be fully removed in 3.0.
```

Comment:
> The configuration option will be fully removed in 3.0.

Actually, the option should remain in 3.0, but only `true` should be accepted (then in 3.1 we can deprecate it, and remove in 4.0)

Reply:
Fair enough, has been adjusted.